logicrunner/goplugin/goplugintestutils/utils.go

```diff
@@ -158,19 +160,20 @@ func (cb *ContractsBuilder) Build(ctx context.Context, contracts map[string]stri
 	}
 
 	for name := range contracts {
-		log.Debugf("Building plugin for contract %q in %q", name, cb.root)
+		logger.Debug("Building plugin for contract ", name, " in ", cb.root)
+
 		err := cb.plugin(name)
 		if err != nil {
 			return errors.Wrap(err, "[ Build ] Can't call plugin")
 		}
-		log.Debugf("Built plugin for contract %q", name)
+		logger.Debug("Built plugin for contract ", name)
 
 		pluginBinary, err := ioutil.ReadFile(filepath.Join(cb.root, "plugins", name+".so"))
 		if err != nil {
 			return errors.Wrap(err, "[ Build ] Can't ReadFile")
 		}
 
-		log.Debugf("Deploying code for contract %q", name)
```

Comment:
return back "%q" flag, please. don't overuse (...) logging

Reply:
Why? it's really way better than using formats.